analyzers/src/ITs.JsonParser/Reports/InputReport.cs

```diff
@@ -42,7 +42,8 @@ public InputReport(string path, JsonSerializerOptions options)
             ? (fileName.Substring(0, index), fileName.Substring(index + 1))
             : (fileName, null);
 
-        Project = Path.GetFileName(Path.GetDirectoryName(path));
+        var projectDirectory = Directory.GetParent(Path.GetDirectoryName(path)).FullName;
```

Comment:
Using `DirectoryInfo` is a more expensive wrapper around these basic methods from `Path` class
```suggestion
        var projectDirectory = Path.GetDirectoryName(Path.GetDirectoryName(path));
```
so then, you can go just 
```suggestion
Project = Path.GetFileName(Path.GetDirectoryName(Path.GetDirectoryName(path)));
```

Reply:
Anyway, please throw it away and add `string project` parameter, because we know the value from our previous processing